virtual_machine/windows/main.tf

```diff
@@ -74,3 +74,81 @@ resource "azurerm_virtual_machine" "virtual_machine" {
     builtby = "Terraform"
   }
 }
+
+resource "azurerm_metric_alertrule" "metric_alertrule_cpu" {
+  count               = "${var.number_servers}"
+  name                = "${var.application_name}_${var.hostname}_vm_${count.index + var.start_index}-cpu"
```

Comment:
Should this portion that's hard-coded where it says "_vm_" be the "metric_alertrule" from the resource type?

Reply:
fixing name standard issues.